doc/user_guide/user_scripts.rst

```diff
@@ -102,9 +104,11 @@ is applied). These are:
     # List of all files that psyclone will skip processing
     FILES_TO_SKIP = ["broken_file1.f90", "broken_file2.f90"]
 
-    # Boolean to decide whether PSyclone should chase external modules while
-    # creating a PSyIR tree in order to obtain better external symbol information.
-    # It can also be a list of module names for more precise control
+    # Whether to chase the imported modules to improve symbol information (it can
+    # also be a list of module filenames chase only specific modules). This has to
+    # be used in combination of '-I' command flag in order to point to the module
+    # location directory and we strongly suggest using it in combination with the
```

Comment:
"...directory. We also strongly recommend using the '--enable-cache' flag to reduce the performance overhead."

Reply:
Done